### PR TITLE
Clients

### PR DESCRIPTION
**Pull request by jacopobacci**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,7 +61,7 @@ <h4>Pro</h4>
             <a href="#"><button>Get Started</button></a>
           </div>
           <div class="pricing__card">
-            <img src="assets/img/pro.svg" alt="enterprise" />
+            <img src="assets/img/enterprise.svg" alt="enterprise" />
```
Comment (glgoose) [inline index.html:64]: this was already fixed in #30
Did you commit directly to main then?

File: index.html [hunk 2]
```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
           </div>
         </div>
       </section>
+
+      <section class="clients">
+        <div class="clients__testimonials">
+          <h4>Testimonials</h4>
```
Comment (glgoose) [inline index.html]: should be `h3` , the tags are for semantics and we can't skip h3 by going from h2 to h4 directly, we can always change the styling with css

File: index.html [hunk 3]
```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
...
+          <h2>What Clients Says About Us</h2>
+          <hr />
```
Comment (glgoose) [inline index.html:78]: lin 78 can be deleted right? can't find the opening tag

File: index.html [hunk 4]
```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
...
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus eligendi tenetur voluptatum, quas consequuntur
+            consectetur.
+          </p>
+          <div class="clients__arrows">
+            <span>&LeftArrow;</span>
+            <span>&RightArrow;</span>
+          </div>
+        </div>
+        <div class="clients__card">
+          <img src="" alt="" />
```
Comment (glgoose) [inline index.html]: image src & alt missing

File: index.html [hunk 5]
```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
...
+          <q>Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam ducimus labore asperiores, vel ab ipsum!</q>
+          <div class="clients__card-bottom">
+            <i class="lni lni-quotation"></i>
+            <img src="" alt="" />
```
Comment (glgoose) [inline index.html]: img src & alt missing

**Review by glgoose — CHANGES_REQUESTED** [review]
remove `assets/img/.DS_Store` and maybe check globpattern for `.gitignore` again to see how we can avoid adding these files

**Review by JimCarlier — COMMENTED** [review]
checked and found no additionnal changes

File: index.html (hunk 1 above)
Comment (jacopobacci) [reply index.html:64]: its weird because in my local file there's the right name, let's see at the next push

File: index.html (hunk 3 above)
Comment (jacopobacci) [reply index.html:78]: it is an horizontal break as in the mockup there is this kind of separator

File: index.html (hunk 4 above)
Comment (jacopobacci) [reply index.html]: removed all the tag, don't need it

File: index.html (hunk 5 above)
Comment (jacopobacci) [reply index.html]: cannot find the photo in the img folder thats why i left it blank

File: index.html (hunk 2 above)
Comment (jacopobacci) [reply index.html]: changed, thanks for the explanation

File: index.html (hunk 5 above)
Comment (glgoose) [reply index.html]: could you find an alternative somewhere? double check if it's not in the bxl-johnsons 1st (if not already done)

Comment (jacopobacci) [reply index.html]: ok, alternatively i use a rnd img

Comment (jacopobacci) [reply index.html]: i take a screenshot so i dont waste time on it

**Pull request merged**